### PR TITLE
improve warning for missing RulePolicy

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- add `RulePolicy` to log message
- change `info` log to `warnings` as the user can and should do something about it

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
@ArjaanBuijk Do you have time to review this?

**Comment by ArjaanBuijk** [comment]
@wochinge , I will review this today.

File: rasa/core/policies/ensemble.py [hunk 1]
```diff
@@ -67,10 +68,13 @@ def _check_for_important_policies(self) -> None:
...
-                f"MappingPolicy not included in policy ensemble. Default intents "
-                f"'{USER_INTENT_RESTART} and {USER_INTENT_BACK} will not trigger "
-                f"actions '{ACTION_RESTART_NAME}' and '{ACTION_BACK_NAME}'."
+            rasa.shared.utils.io.raise_warning(
+                f"Neither '{RulePolicy.__name__}' nor '{MappingPolicy.__name__}' "
+                f"(deprecated) are included in the model's policy configuration. "
+                f"Default intents such as '{USER_INTENT_RESTART}' and "
+                f"'{USER_INTENT_BACK}' will not trigger actions "
+                f"'{ACTION_RESTART_NAME}' and '{ACTION_BACK_NAME}'.",
+                docs=DOCS_URL_ACTIONS,
```
Comment (ArjaanBuijk) [inline rasa/core/policies/ensemble.py:77]: The URL points to https://rasa.com/docs/rasa/actions.
Maybe it should point to https://rasa.com/docs/rasa/default-actions ?

**Review by ArjaanBuijk — APPROVED** [review]
It is all clear and working. I was just wondering if the URL link could be more targeted to the default_actions.

**Pull request merged**